README.md

```diff
@@ -5,7 +5,7 @@
 - [discord.py](https://github.com/rapptz/discord.py)
 - lots of Python modules
 
-[![NotSoSuper's Dev](https://discordapp.com/api/guilds/178313653177548800/widget.png?style=banner2)](https://discord.gg/QQENx4f)
+[![NotSoServer](https://discordapp.com/api/guilds/178313653177548800/widget.png?style=banner2)](https://discord.gg/QQENx4f)
```

Comment:
Did you mean https://discord.gg/russia ?

Reply:
No, https://discord.gg/staff